### PR TITLE
Updates to superflux extractor (e.g RegTest, invalid vals, etc.)

### PR DESCRIPTION
**Pull request by cvf-bcn-gituser**
Added Regression Tests based on two audio files, whose values were captured on previous runs with the same parameters.
Tested on Ubuntu and Linux Subsystem on Windows.
Superfluxextractor covers peaks and novelty tests internally.
Dummies added  to maintain 1:1 correspondence between src, file  and UT.

### CODE REVIEW

File: test/src/unittests/rhythm/test_superfluxextractor.py [hunk 1]
```diff
@@ -17,12 +17,60 @@
...
+            for j in range(1,len(expected)):
+                if found[i] <= expected[j] and found[i] >= expected[j-1]:
+                    if fabs(found[i] - expected[j-1]) < fabs(expected[j] - found[i]):
+                        self.assertAlmostEqual(found[i], expected[j-1], precision)
+                    else:
+                        self.assertAlmostEqual(found[i], expected[j], precision)
+
     def testSilence(self):
         # zeros should return no onsets (empty array)
         self.assertEqualMatrix(SuperFluxExtractor()(zeros(44100)), [])
```
Comment (dbogdanov) [inline test/src/unittests/rhythm/test_superfluxextractor.py:76]: The output is 1-D vector, so we can use `assertEqualVector` instead.

File: test/src/unittests/rhythm/test_superfluxextractor.py [hunk 2]
```diff
@@ -55,7 +103,6 @@ def testImpulse(self):
         precission = (hopSize + frameSize) / sampleRate
```
Comment (dbogdanov) [inline test/src/unittests/rhythm/test_superfluxextractor.py:103]: MIsspel: `precision`

**Review by dbogdanov — CHANGES_REQUESTED** [review]
There's a redundant call `self.assertComputeFails(Onsets(), array([[]]), [])`, possibly left from another test.

File: test/src/unittests/rhythm/test_superfluxextractor.py (hunk 1 above)
Comment (cvf-bcn-gituser) [reply test/src/unittests/rhythm/test_superfluxextractor.py:76]: self.assertEqualMatrix should use self.assertEqualVector

**Pull request merged**